jaxrs-api/src/main/java/jakarta/ws/rs/core/NewCookie.java

```diff
@@ -249,7 +267,9 @@ public NewCookie(final Cookie cookie, final String comment, final int maxAge, fi
      * @param sameSite specifies the value of the {@code SameSite} cookie attribute
      * @throws IllegalArgumentException if cookie is {@code null}.
      * @since 3.1
+     * @deprecated This constructor will be removed in a future version. Please use {@link NewCookie.Builder} instead.
```

Comment:
Just noticed this, but since this constructor was added in 3.1, we should probably just delete it instead of deprecating it in the same release that we introduce it.

Reply:
Well, that's is my opinion too but at the time @mkarg did not share it https://github.com/eclipse-ee4j/jaxrs-api/pull/895#issuecomment-679835532

Reply:
:) This PR has been open for a while! I'd forgotten that conversation.  I'm fine with it as-is.  Thanks again!

Reply:
We decided to just deprecate so the users have more time to adopt the new solution before we force them to do that.